Spec/RealtimeClientConnection.swift

```diff
@@ -2878,8 +2878,8 @@ class RealtimeClientConnection: QuickSpec {
                         }
 
                         waitUntil(timeout: testTimeout) { done in
-                            // Renewal will fail
-                            client.connection.once(.failed) { stateChange in
+                            // Renewal will lead to a disconnect (again)
+                            client.connection.once(.disconnected) { stateChange in
```

Comment:
Should this test wait and see if a new token comes?

Reply:
Well, a new valid token will never come in, given the way this test is designed. Or am I misunderstanding your question?

Reply:
Sorry, I misunderstood the test.
It looks good.